job/server/src/main/java/alluxio/master/job/plan/PlanTracker.java

```diff
@@ -105,8 +105,8 @@ public PlanTracker(long capacity, long retentionMs,
     Preconditions.checkArgument(retentionMs >= 0);
     mRetentionMs = retentionMs;
     mMaxJobPurgeCount = maxJobPurgeCount <= 0 ? Long.MAX_VALUE : maxJobPurgeCount;
-    mCoordinators = new ConcurrentHashMap<>(0,
-        0.95f, ServerConfiguration.getInt(PropertyKey.MASTER_RPC_EXECUTOR_PARALLELISM));
+    mCoordinators = new ConcurrentHashMap<>(0, 0.95f,
```

Comment:
Move the comment/description from the property key here so we keep why the `Math(8, 2*nCPU)` here?

Reply:
@yuzhu Why did we relied on this property in the first place?

Reply:
Should really make a new property, but everyone hates that..